gym/eval.sh

```diff
@@ -0,0 +1,2 @@
+chmod 777 -Rf tf_od_api/workspace/train_core/
+nvidia-docker run -it -v $(pwd)/tf_od_api/workspace/train_core:/home registry.fsoc.hpi.uni-potsdam.de/coral-ai > tf_od_api/workspace/train_core/eval.log
```

Comment:
You run something on `$(pwd)/tf_od_api/...` but then you redirect the output to only `tf_od_api/...`. You should probably have the `pwd` on both sides